Sources/SwiftPlot/GraphLayout.swift

```diff
@@ -363,6 +365,21 @@ public struct GraphLayout {
                               angle: 0)
         }
     }
+
+    mutating func addAnnotation(annotation: Annotation) {
```

Comment:
@WilliamHYZhang you have declared this function in GraphLayout so it will be inaccessible to any plot. Move it to 
`extension Plot where Self: HasGraphLayout {` .
This is an extension for `Plot` so each plot will have access to this function.